tests/unit/test_expression_tree/test_interpolant.py

```diff
@@ -87,6 +87,11 @@ def test_interpolation_2_x_2d_y(self):
         np.testing.assert_array_almost_equal(
             interp.evaluate(y=np.array([0, 0])), 0, decimal=3
         )
+        # cubic
+        interp = pybamm.Interpolant(x, z, (var1, var2), interpolator="cubic")
+        np.testing.assert_array_almost_equal(
+            interp.evaluate(y=np.array([0, 0])), 0, decimal=3
+        )
```

Comment:
I know you just copied the linear test but we really need a better test for this :P

Reply:
Yeah could check against z for a point in a finer interpolation